module/canvas/token-placement.mjs

```diff
@@ -111,6 +111,7 @@ export default class TokenPlacement {
         obj.eventMode = "none";
         const placement = await this.#requestPlacement();
         if ( placement ) placements.push(placement);
+        else this.#previews[this.#currentPlacement]?.object.clear();
```

Comment:
Since we already have the object as a local variable above, we can just use that to clean things up.

```suggestion
        else obj.clear();
```